flatsurf_algorithms.tex

```diff
@@ -828,7 +828,7 @@ \subsection{$\GL(2,\bR)$-action and linear manifolds}
 
 \begin{proof}
 Just use the fact that the real projection $x+iy \mapsto x$ and imaginary projection
-$x+iy \mapsto y$ live the tangent space invariant (since they are limit of $\GL(2,\bR)$
```

Comment:
What did you mean to write here?

Reply:
You are not allowed to apply an element of `Mat(2, R)` to a translation surface but only elements in `GL(2, R)`. However, infinitesimally (in the tangent space) that makes perfect sense. The way to proceed is to write down the (derivative) of the action on the tangent bundle. It is just the simple formula I wrote somewhere. And it is clear that this action extends continuously to `Mat(2, R)`.

Reply:
Sorry, the grammar just doesn't work out in the original phrase. Did you mean "live in" or "lift"?

Reply:
"leave the tangent space invariant"

Reply:
lol, that makes a lot of sense. I thought of "the invariant of the tangent space" (whatever that might have been) and could not make any sense of it.

Reply:
Fixed. Thanks @slel.